docker-compose.memory-leak.yml

```
@@ -5,5 +5,12 @@ services:
     volumes:
       - .:/app
+      - /app/node_modules
     init: true
-    shm_size: '256m'
+    shm_size: '1gb'
+    healthcheck:
+      interval: 10s
+      timeout: 5s
+      retries: 3
+      start_period: 30s
+      test: [ "CMD", "chromium-browser", "--version" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)